schema/deploy/computed_columns/form_change_calculated_ei_performance.sql

```diff
@@ -0,0 +1,25 @@
+-- Deploy cif:computed_columns/form_change_calculated_ei_performance to pg
+
+begin;
+
+create or replace function cif.form_change_calculated_ei_performance(fc cif.form_change)
+returns numeric
+as
+$fn$
+
+  select
+    case
+      when (fc.new_form_data->>'baselineEmissionIntensity' is null)
+        or (fc.new_form_data->>'targetEmissionIntensity' is null)
+        or (fc.new_form_data->>'postProjectEmissionIntensity' is null)
+        or (fc.new_form_data->>'baselineEmissionIntensity' = fc.new_form_data->>'targetEmissionIntensity')
```

Comment:
should the case `or (fc.new_form_data->>'baselineEmissionIntensity' = fc.new_form_data->>'targetEmissionIntensity')` return a non-none value?

Reply:
If baseline and target are equal, that produces dividing by 0, and I've always seen this handled using `null`. Is there something else that's better to return?

Reply:
Should we test
```
(fc.new_form_data->>'baselineEmissionIntensity')::numeric - (fc.new_form_data->>'targetEmissionIntensity')::numeric) != 0
```
instead?